freetextresponse/templates/freetextresponse_view.html

```diff
@@ -23,4 +25,9 @@ <h2 class="problem-header">{{ display_name }}</h2>
         <div class="used-attempts-feedback" aria-live="polite">{{ used_attempts_feedback }}</div>
     </div>
     <div class="capa_alert user_alert">{{ user_alert }}</div>
+    <ul class="responses-box" id="responses-box">
+        <button class="hide-button">Hide</button>
+        <p class="responses-title">Submissions by others</p>
+        <li class="response-list"></li>
```

Comment:
Why are we using `<ul>` and `<li>` elements for displaying responses?

Especially since the "list" actually seems to be contained within a list item?